### ISSUE TITLE
The Session Start Protocol reports every piece of state it manages and never its own version, so an install can run a fixed defect for weeks

### ISSUE THREAD

**Issue by agerescue** [rebelytics/one-skill-to-rule-them-all#199]
**Target:** `SKILL.md` — Session Start Protocol, steps 1-7.
**Verified at:** upstream `main` (`80ab2f3`) and the v3.3.1 tag (`b20625d`). Nothing in
the core or in `references/environments.md` reads an install's provenance file.
**Adjacent, not duplicate:** #127 is the same blind spot at a different step — it is
about what a review may *edit* (category (a) vs (b)); this is about what the session is
*running*. #185 (the activation tier ladder not degrading) is about a tier disappearing,
not about the version of the skill that reads it.

## What happened

An install pinned at a v3.0.1-era commit ran daily for nineteen days. During that window
four defects observed on it were reported upstream, accepted, fixed and released. The
sessions that read the release notifications were still executing the old text — one of
them still hit a zsh glob abort in Session Start step 6, which the same release had
fixed.

The drift surfaced because a human opened an email and asked whether the local copy had
the fix. Nothing in the loop would have raised it. The install carried a provenance file
in the skill directory the entire time:

```json
{ "repo": "…/one-skill-to-rule-them-all.git", "commit": "510caad…", "installed_by": "…" }
```

Steps 1-7 check storage, the frontmatter scan, the review trigger, the activation tiers,
concurrency, `skill:` target resolution and staged updates — every piece of state the
skill manages. What the skill *is* is not among them, so the version is the one fact the
start-up report cannot be wrong about, because it never states it.

## Suggested fix

1. **One line in the start report.** Read whichever provenance the install happens to
   have — `.source.json`, a bundle manifest, a `version:` in frontmatter, the plugin
   manifest — and print the installed version beside the other start-up lines. No network
   call is needed or wanted: the value alone lets a human compare it against a release
   note, which is exactly how this was eventually found.

2. **An upstream report is not resolved when it is accepted.** Where an observation's fix
   lands in a skill the user does not maintain, the entry is resolved when the *local
   install carries the fix*, not when the issue closes. `parked` with a `parked_until:`
   naming the installed version is the honest status — and it fits 3.3.0's rule that a
   park condition names the result in the target rather than the state of a vehicle such
   as a pull request.

## Principle

A self-check enumerates what a tool manages and never what the tool is, so the tool's own
version drift is structurally invisible to it no matter how thorough the check gets.
Wherever an upstream fix does not itself change the downstream artefact, reporting,
fixing and releasing can all succeed while the defect keeps running — resolution happens
at re-install, and nothing observes a re-install. So the first line of a self-check
should be which copy is running.

**Comment by agerescue** [comment]
Correction to the report above.

The sentence "one of them still hit a zsh glob abort in Session Start step 6, which the
same release had fixed" is wrong. In 3.0.1, step 6 is prose with no shell snippet. The
command that aborted under zsh was one I wrote myself while carrying out step 6 (an
unguarded glob over a plugins directory), not anything the skill ships. It is not
evidence about the skill, so please disregard it.

The rest of the report does not depend on it. The version drift, the provenance file
nobody reads, and the four fixes released while the install kept running the old text
are all as stated.

One note for the feedback pre-flight, since this is how the error got in: item (3) asks
whether a problem still exists at upstream HEAD. It does not ask whether the failing line
was upstream's code in the first place.
